test/e2e_flags.go

```diff
@@ -99,6 +100,8 @@ func initializeServingFlags() *ServingEnvironmentFlags {
 		"Set this flag to add extra HTTP request headers sent to the testing deployed KServices. "+
 			"Format: -request-headers=key1,value1,key2,value2")
 
+	flag.StringVar(&f.IngressClass, "ingress-class", "",
```

Comment:
Q. We can use the flag for one Ingress, but how will you do when we implement the websocket Kourier and Istio, for example?

Only Kourier will work with the following code, but how to add multiple ingresses? We will use regex?

```
 if strings.Contains(test.ServingFlags.IngressClass, "kourier") {
```

Reply:
I think we can make an "or“ condition like
``` 
if strings.Contains(test.ServingFlags.IngressClass, "kourier")  || strings.Contains(test.ServingFlags.IngressClass, "istio") {
```

Or, we can define an allowed_ingress_list, and check if test.ServingFlags.IngressClass is in this list. 

Does it make sense?

Reply:
Ah, yes. Sorry I had a misunderstanding when I asked.